### PR TITLE
navi group class added

### PR DESCRIPTION
**Pull request by Sabadi10**
// Thank you

### CODE REVIEW

**Comment by camilorojas8** [comment]
you are welcome

**Comment by AnnaChupina** [comment]
Hello!

**Review by emirayral — APPROVED** [review]
Looks good to me

File: src/Larbi/Atman.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Larbi;
+
+public class Atman {
+
+    public static void main(String[] args) {
+        // Music
```
Comment (AnnaChupina) [inline src/Larbi/Atman.java:6]: //What music?

Comment (AnnaChupina) [reply src/Larbi/Atman.java:6]: HardRock!!

Comment (Sabadi10) [reply src/Larbi/Atman.java:6]: // Raphael said just write something @AnnaChupina

Comment (erdogansezgin58) [reply src/Larbi/Atman.java:6]: Good job !

**Review by erdogansezgin58 — APPROVED** [review]

File: src/Larbi/Atman.java
Comment (Simona888) [inline src/Larbi/Atman.java:1]: Hello World :-)

File: src/Larbi/Atman.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package Larbi;
+
```
Comment (pheonixzee) [inline src/Larbi/Atman.java:2]: boo

File: src/Larbi/Atman.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+public class Atman {
```
Comment (ATM192) [inline src/Larbi/Atman.java:3]: You need to change this class name

**Review by pheonixzee — APPROVED** [review]

**Review by AnnaChupina — APPROVED** [review]

**Pull request merged**